config/repos.yml

```diff
@@ -40,6 +40,9 @@ repos:
     index: 'https://github.com/X-Raym/REAPER-ReaScripts/raw/master/index.xml'
     default: true
     disporder: 12
+  -
+    link: 'https://github.com/ArchieScript/Archie_ReaScripts'
+    index: 'https://raw.githubusercontent.com/ArchieScript/Archie_ReaScripts/master/index.xml' 
```

Comment:
Greetings! Changed the URL of the index as you said. It seems everything is properly done.